pkgs/by-name/ae/aegisub/package.nix

```diff
@@ -78,43 +68,36 @@ stdenv.mkDerivation (finalAttrs: {
     wxGTK
   ];
 
-  buildInputs = [
-    boost
-    expat
-    ffmpeg
-    ffms
-    fftw
-    fontconfig
-    freetype
-    fribidi
-    glib
-    harfbuzz
-    icu
-    libGL
-    libGLU
-    libX11
-    libass
-    libiconv
-    libuchardet
-    pcre
-    wxGTK
-    zlib
-  ]
-  ++ lib.optionals alsaSupport [ alsa-lib ]
-  ++ lib.optionals openalSupport [
-    (if stdenv.hostPlatform.isDarwin then OpenAL else openal)
-  ]
-  ++ lib.optionals portaudioSupport [ portaudio ]
-  ++ lib.optionals pulseaudioSupport [ libpulseaudio ]
-  ++ lib.optionals spellcheckSupport [ hunspell ]
-  ++ lib.optionals stdenv.hostPlatform.isDarwin [
-    AppKit
-    Carbon
-    Cocoa
-    CoreFoundation
-    CoreText
-    IOKit
-  ];
+  buildInputs =
+    [
+      boost
+      expat
+      ffmpeg
+      ffms
+      fftw
+      fontconfig
+      freetype
+      fribidi
+      glib
+      harfbuzz
+      icu
+      libGL
+      libGLU
+      libX11
+      libass
+      libiconv
+      libuchardet
+      pcre
+      wxGTK
+      zlib
+    ]
+    ++ lib.optionals alsaSupport [ alsa-lib ]
+    ++ lib.optionals openalSupport [
+      (if stdenv.hostPlatform.isDarwin then darwin.apple_sdk.frameworks.OpenAL else openal)
+    ]
```

Comment:
This is actually kind of subtle (and you might want to extract this into its own PR for clarity):

After the refactor, on Darwin, OpenAL is always present, so this part can simply be:

```
++ lib.optionals (openalSupport && !stdenv.hostPlatform.isDarwin) [
  openal
]
```

But in order to properly disable openal on Darwin, we now need

```
cmakeFlags = [ "-DWITH_OPENAI=OFF" ];
```

when `openalSupport` is false.

Reply:
I like the idea of dropping this in its own PR for clarity. That way we can document the change properly.

Reply:
I had to move the patch into here due to conflicts with the new default SDK